commands/balance.js

```diff
@@ -36,10 +36,13 @@ module.exports = function container (get, set, clear) {
             if (err) return cb(err)
             s.exchange.getQuote(s, function (err, quote) {
               if (err) throw err
-              var bal = moment().format('YYYY-MM-DD HH:mm:ss').grey + ' ' + engine.formatCurrency(quote.ask, true, true, false)
-              bal += ' ' + (s.product_id + ' Asset: ').grey + balance.asset.white + ' Currency: '.grey + balance.currency.yellow + ' Total: '.grey + n(balance.asset).multiply(quote.ask).add(balance.currency).value().toString().yellow
+              
+              var bal = moment().format('YYYY-MM-DD HH:mm:ss').grey + ' ' + engine.formatCurrency(quote.ask, true, true, false) + (s.product_id).grey + '\n'
```

Comment:
correct to `var bal = moment().format('YYYY-MM-DD HH:mm:ss').grey + ' ' + engine.formatCurrency(quote.ask, true, true, false) + ' ' + (s.product_id).grey + '\n'`

need space between quote.ask and product_id

Reply:
Yup, good catch.